com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Parser/HrJobRoles.php

```diff
@@ -148,7 +148,11 @@ function summary(&$values) {
     $session = CRM_Core_Session::singleton();
     $dateType = $session->get('dateTypes');
 
-    $contractDetails = CRM_Hrjobcontract_BAO_HRJobContract::checkContract($params['job_contract_id']);
+    $contractDetails = NULL;
```

Comment:
All the changes in this file seem OK, but they were not suggested during the PR review. Maybe they should be placed in another commit?